src/main/webapp/WEB-INF/views/training.jsp

```diff
@@ -87,6 +87,7 @@
                     }
                 });
 
+
```

Comment:
If possible no unneeded line changes (but not that important here, since the whole file is edited)